drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -298,12 +299,16 @@ function DragAndDropBlock(runtime, element, configuration) {
                     input.class_name = item_user_state.correct_input ? 'correct' : 'incorrect';
                 }
             }
+            var content_html = item.displayName;
+            if (item.backgroundImage) {
+                content_html = '<img src="' + item.backgroundImage + '" alt="' + item.backgroundDescription + '" />';
```

Comment:
I know this was already coded this way, but it would be better to just pass `displayName`, `backgroundImage`, and `backgroundDescription` to the template and move this conditional logic and HTML generation there (`view.js`).

Also, `item.backgroundDescription` needs to be HTML escaped - you'll get that for free with the move to view.js.